example/lib/features/gridview/gridview_fixed_height_demo/gridView_fixed_height_demo_page.dart

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
The file name 'gridView_fixed_height_demo_page.dart' isn't a snake_case identifier.
Try changing the name to follow the snake_case style.

Reply:
okay, i'll change the name to ‘grid_view_fixed_height_demo_page’